querydsl-jpa/src/main/java/com/querydsl/jpa/NativeSQLSerializer.java

```diff
@@ -57,7 +57,11 @@ protected void appendAsColumnName(Path<?> path, boolean precededByDot) {
         if (path.getAnnotatedElement().isAnnotationPresent(Column.class)) {
             SQLTemplates templates = getTemplates();
```

Comment:
These templates are only used in one branch of the if block, so you *could* move them in there.